pages/index.js

```diff
@@ -230,15 +232,15 @@ export default function Playground() {
                         />
                     </div>
                     <div className="mb-2">
-                        <label className="block" htmlFor="startWeekOnClassName">
-                            Start Week On
```

Comment:
Moved "Start Week On" under the newly added "Toggle Class" input so the formatting of the inputs here looks a little better.